_docs/pages/instrumentation/common-use-cases.md

```diff
@@ -253,3 +253,69 @@ if request.get('debug'):
         tags={tags.SAMPLING_PRIORITY: 1}
     )
 ```
+
+### Tracing Messaging Scenarios
+
+There are two messaging styles that should be handled, Message Queues and Publish/Subscribe (Topics).
+
+The main distinction between these styles is that a message written to a queue can only be consumed by **at most** one consumer, whereas a message published on a topic can be consumed by zero or more subscribers. The other point to note is that although a message can be sent/published by a producer, that does not guarantee that it will be consumed - messaging systems are asynchronous and therefore the producer has no visibility of when or if the message was delivered.
+
+From a tracing perspective, the messaging style is not important, only that the span context associated with the producer is propagated to the zero or more consumers of the message. It is then the responsibility of the consumer(s) to create a span to encapsulate processing of the consumed message and establish a _FollowsFrom_ reference to the propagated span context.
```

Comment:
Might be worth having the distinction, but do you have some concrete scenarios where a new trace would be preferred rather than same trace?